### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by the98** [emesene/emesene#699]
I've downloaded today the latest version from the repository. My OS is Debian Sid, amd64 architecture.

Randomly emesene makes the CPU usage go to 50%, freezing emesene for a few seconds. This usually happens when a contact sends me a response I switch to the contact's window to write a message, however it might also happen doing some other operation, like scrolling on the main window.

Try to see if you can reproduce the bug to see what I mean.

Kind regards.

**Comment by Mystro256** [comment]
I can confirm this as a regression in 2.11.6dev (compared to 2.11.5)

**Comment by Sbte** [comment]
Regression test plx? You do the testing, I do the fixing :D

**Comment by Mystro256** [comment]
I would if I had a reliable way of testing; I'll do a bisect but it'll take me some time...

**Comment by Sbte** [comment]
No problem

**Comment by Mystro256** [comment]
I think I'm gonna give up the bisect, I'm starting to think it's not a regression, but merely coincidence when I tested before. It's very hard to figure it out because it's very inconsistent. Although I did confirm that for whatever reason receiving offline messages, specifically, use a lot of CPU power (e.g. while status is invisible, receive messages from someone else who's online)

As well, as @marianoguerra said in #701, CPU use is probably caused by excessive use of timeout_add
